charts/barman/templates/statefulset.yaml

```diff
@@ -23,18 +23,34 @@ spec:
         - name: barman
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - "/bin/sh"
+                  - "-c"
+                  - "HOME=/home/barman && cp /.pgpass $HOME/ && chown barman:barman $HOME/.pgpass && chmod 600 $HOME/.pgpass"
```

Comment:
I guess there's a reason why this magic is required? If yes, please add a short explanation as a template comment:
```
{{- /*
here goes your explanation
*/ -}}
```

Reply:
Postgresql's [Docs:](https://www.postgresql.org/docs/9.4/libpq-pgpass.html)
> On Unix systems, the permissions on .pgpass must disallow any access to world or group

I couldn't find a better way to chown the file properly. Inputs are welcome @tongpu :) Otherwise I'll add the comment.

Reply:
It should be possible to configure `fsGroup` for the secret volume mount.

Reply:
It is possible, but group doesn't help as mentioned above, or am I missing something?

Reply:
And `fsUID` doesn't seem to be widely supported. Just a quick comment to the template why this is required, so that people that ask the same question in the future know why.